CMakeLists.txt

```diff
@@ -213,29 +213,29 @@ if (APPLE)
 endif()
 
 # Load all of our third party directories
-add_subdirectory(thirdparty)
+add_subdirectory(thirdparty SYSTEM)
```

Comment:
Since this line requires cmake version 3.25, regardless of if this is merged or not, i believe the minimum required version should be updated to require 3.25 to ensure no weird build issues occur on older clients

Reply:
Yeah. I can't think of another way to accomplish this, so I've bumped up the required version.